### DISCUSSION TITLE
Settings of QRCODE which is in README.md

### DISCUSSION THREAD

**Discussion by markas** [chillerlan/php-qrcode#103] [Q&A] [answered]
Hello,

I saw very nice QRCODE example: 

https://raw.githubusercontent.com/chillerlan/php-qrcode/main/.github/images/example.svg

Can somebody share settings how create same one (colors, size, white dots and other)? 

Thank you!

**Comment by codemasher** [comment, marked as answer]
Hi, the example being used is from the current development branch [over here](https://github.com/chillerlan/php-qrcode/blob/main/examples/svg.php), it's not possible out of the box with the recent v4.x releases and below, but could be adapted to work with it. See: https://github.com/chillerlan/php-qrcode/blob/main/src/Output/QRMarkup.php#L102-L229

**Reply by markas** [reply]
codemasher, I have one issue: If I use: 'drawCircularModules' => true, then I get: https://www.dropbox.com/s/c4ow44ji3scwk18/Screenshot%202021-12-14%20at%2020.31.25.png?dl=0

If use: 'drawCircularModules' => false, I get: https://www.dropbox.com/s/0jfakj0ebqhxdze/Screenshot%202021-12-14%20at%2020.31.42.png?dl=0

All settings same like in svg.php example file. Do you have any idea why drawCircularModules not working?

**Reply by codemasher** [reply]
Something's certainly not right but i can't tell exactly what without the code you're using. Are you running the `dev-main` branch or have you ported the SVG module to an earlier relase?

**Reply by codemasher** [reply]
Also, what's the generated source of the SVG? And, in doubt, the browser you're using?
